core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -254,10 +256,32 @@ export class CustomizeInteractionModalComponent
   }
 
   isSaveInteractionButtonEnabled(): boolean {
-    return !!(
-      this.hasCustomizationArgs &&
-      this.stateInteractionIdService.displayed &&
-      (this.getCustomizationArgsWarningsList().length === 0));
+    let explorationTitle = this.getTitle(
+      this.stateInteractionIdService.displayed);
+    let EndExploration = 'End Exploration';
```

Comment:
This should be a constant in the app constants file.

Reply:
I tried finding this as a constant in app constants file, but I think it is not defined as a constant there!

Reply:
for the front-end test thing, now I have removed the check `if (INTERACTION_SPECS[interactionId] !== undefined)` because now no test is failing on removing this. I faced some issue sometime before but now everything seems fine on removing this check!

Reply:
done